fix(docs): clarify data layout of bounced messages (#3472)
tact-lang/tact: docs/src/content/docs/book/bounced.mdx

message BB {}
```

Since message fields are always laid out sequentially and no automatic rearrangements are made, the partial representation made by the `bounced<M>{:tact}` constructor would only contain the first fields that fit entirely within the 224-bit limit.

All subsequent fields that exceed the limit will not be available in bounced message receivers. This also includes fields that fit only partially.

> If the first field does not wholly fit the 224-bit limit, none of the bounced message fields would be available. Therefore, prefer arranging the important fields that fit into the limit first.

Author: ```suggestion
If the first field does not wholly conform to the 224-bit limit, none of the bounced message fields would be available. Therefore, to ensure access, prefer arranging the important fields first.
```

Due to excessive use of the "fit" elsewhere, I thought that it would be better to replace its use here with a synonym.

}
```

:::caution

>   To prevent potential issues, Tact only allows using the fields that wholly fit the limit. However, the blockchain views that limit as a truncation boundary and keeps the raw bits that fit the total 256-bit message limit in the original message body cell.

Reviewer: It's confusing: need to clarify 256 vs 224 bits

Author: <img width="1327" height="214" alt="image" src="https://github.com/user-attachments/assets/e04ad6d8-440d-42a8-8f74-4963c94bd625" />

Earlier there's an explanation for the "256 vs 224 bits" discrepancy. We can link the phrase "total 256-bit message limit" to the start of the "Caveats".

Author: ```suggestion
  To prevent potential issues, Tact only allows using the fields that wholly fit the limit. However, the blockchain views that limit as a truncation boundary and keeps the raw bits that fit the [total 256-bit body limit](#caveats) in the original bounced message body cell.
```

Author: I've applied the changes above :)
